### PR TITLE
Trigger fixes for 2022

### PR DESCRIPTION
**Pull request by haddadanas**
This PR:

- Removes some triggers not needed/used anymore.
- Documents used trigger filters for each trigger
- Corrects the Trigger bits of `HLT_VBF_DoubleMediumDeepTauPFTauHPS20_eta2p1`
- Change `HLT_Ele32_WPTight_Gsf` to `HLT_Ele30_WPTight_Gsf`
- Adds the `cross-tau-jet` trigger to the selection

### CODE REVIEW

**Comment by haddadanas** [comment]
The filter bits in the Trigger legs of jets was changed to 1, which corresponds to the jet `hlt4PixelOnlyPFCentralJetTightIDPt20` filter (probably the loosest filter for jets). Should it be reversed to `None`?
It is unclear what None does, but the change didn't change anything statistic-wise

**Comment by riga** [comment]
Is this PR ready for review/merging?

**Review by riga — APPROVED** [review]

**Comment by haddadanas** [comment]
@riga yes it is ready for merging, just one question:
Should we keep it like this with the `TriggerLegs` of the Jets?

File: hbt/tasks/studies/triggers.py [hunk 1]
```diff
@@ -17,19 +18,36 @@
...
+    Base task for trigger related studies.
     """
 
     sandbox = dev_sandbox(law.config.get("analysis", "default_columnar_sandbox"))
 
     version = None
 
+    lfn_indices = law.CSVParameter(
+        cls=luigi.IntParameter,
+        default=0,
```
Comment (riga) [inline hbt/tasks/studies/triggers.py:35]:
```suggestion
        default=(0,),
```

**Review by riga — APPROVED** [review]

**Review by riga — APPROVED** [review]

**Pull request merged**